.github/workflows/qa-automation.yml

```diff
@@ -1,39 +1,32 @@
 name: Automate QA
 
 on:
-  pull_request:
+  pull_request_target:
     types:
       - review_requested
-    branches: [development]
+    branches: [development, main]
 
 jobs:
   add-to-project:
+    env:
+      ADD_TO_PROJECT_PAT: ${{ secrets.ADD_TO_PROJECT_PAT }}
     name: Add issue to project
     runs-on: ubuntu-latest
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Authenticate as UbiquiBot
-        uses: tibdex/github-app-token@v1.7.0
-        id: get_installation_token
-        with:
-          app_id: ${{ secrets.UBIQUITY_BOUNTY_BOT_APP_ID }}
-          private_key: ${{ secrets.UBIQUITY_BOUNTY_BOT_PRIVATE_KEY }}
+    permissions:
+      contents: read
 
-      - name: add pr to project
-        if: contains(github.event.pull_request.requested_reviewers.*.login, 'Draeieg')
```

Comment:
This is actually needed! would you re-add this check?

Reply:
fixed